### ISSUE TITLE
[Bug] New Tab Page/Home Page (Default) - CPU spike for a long time

### ISSUE THREAD

**Issue by gunir** [Floorp-Projects/Floorp#523]
### Attention!

- [X] I understand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

Floorp with mostly default settings.

High CPU on startup
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/6a627918-2303-4c4b-8603-73b574fad067)

Homepage Settings:
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/7a71f609-6120-4489-92f0-ea84243052f1)

List of all addons:
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/a5b8f69d-0a03-47b1-a6a6-9c181a2ffca8)

Video: https://streamable.com/hgm446

### To Reproduce

1. Open Task Manager
2. Start Floorp
3. Check CPU Usage

1. Open a homepage new tab
2. Check CPU Usage

Even with all addons disabled, the problem still persist unless switching to Private Browsing mode.

### Expected behavior

CPU Usage shouldn't spike that long.

### Screenshots

High CPU on startup
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/6a627918-2303-4c4b-8603-73b574fad067)

### OS Info

Windows 11

### Floorp Version

11.4.1

### Additional context

Honestly I have no idea why this happened, only thing to keep in mind that  switching to Private Browsing fixed this issue

**Comment by surapunoyousei** [comment]
Did you try to disable blur option?
![image](https://github.com/Floorp-Projects/Floorp/assets/73892113/32d9f5ba-8439-4ad8-972d-044a2983b7c8)

**Comment by gunir** [comment]
> Did you try to disable blur option? ![image](https://user-images.githubusercontent.com/73892113/277341679-32d9f5ba-8439-4ad8-972d-044a2983b7c8.png)

Yeah, disabling Blur reduces CPU usage by 5-7% but the CPU spike still lasts for about 10s, and happen everytime a new tab is opened.

**Comment by cantalupo555** [comment]
I am having the same problem, the function is disabled.

I realize that CPU consumption increases mainly when I'm using the "Browser Manager Siderbar".


![image](https://github.com/Floorp-Projects/Floorp/assets/12485871/1332ab4f-b1f6-49b1-bba3-2a70c12e3835)

![image](https://github.com/Floorp-Projects/Floorp/assets/12485871/ee20598c-2dba-4938-b355-0a58d405084d)
